### DISCUSSION TITLE
Open drawer after hitting fab button to mark all articles read

### DISCUSSION THREAD

**Discussion by Matth7878** [spacecowboy/Feeder#166] [Ideas]
Love the app!

Yet there is one minor annoyance. When you mark all articles read you end up on an empty screen where there is nothing to do. To continue reading articles from others tags the only choice is to manually open the drawer.
It's not a big deal but it would be nice if feeder were to open drawer when hitting fab button.

**Comment by spacecowboy** [comment]
Could be a good addition

**Comment by Matth7878** [comment]
For those interested I created a feature requesting this.
See #375.
